bootstrap

```diff
@@ -14,7 +14,7 @@ if [ -f /etc/debian_version ]; then
     fi
 fi
 if [ -f /etc/redhat-release ]; then
-    for package in python-pip python-virtualenv python-devel python-six libevent-devel libxml2-devel libxslt-devel zlib-devel; do
+    for package in python3-pip python3-virtualenv python3-devel python3-six libevent-devel libxml2-devel libxslt-devel zlib-devel; do
```

Comment:
python3-six wouldn't be needed anymore, it is mostly for backwards compatibility between py2/py3